docs/SlavaUkraini/examples/overlays/index.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: tutorial_v2
+title: Overlays
+---
+
+## Overlays
+
+There are 3 Overlays in Leaflet API: 
+- [`ImageOverlay`](/reference.html#imageoverlay): Raster Layer, Extends [`Layer`](/reference.html#layer)
+- [`VideoOverlay`](/reference.html#videooverlay): Raster Layer, Extends [`ImageOverlay`](/reference.html#imageoverlay)
+- [`SVGOverlay`](/reference.html#svgoverlay): Vector Layer, Extends [`ImageOverlay`](/reference.html#imageoverlay)
+
+In this tutorial, you’ll learn how to use these Overlays.
+
+### `ImageOverlay`
+
+`L.ImageOverlay` is used to load and display a single image over specific bounds of the map. 
+
+To add an image overlay [`L.ImageOverlay`](/reference.html#imageoverlay) use this:
+```
```

Comment:
@KonstantinBiryukov Please make sure there's a space between headings and paragraphs/code blocks, and between code blocks and the next paragraphs in this file.